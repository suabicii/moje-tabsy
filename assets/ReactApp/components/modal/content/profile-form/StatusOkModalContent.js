import React from "react";

function StatusOkModalContent({setIsModalOpen}) {
    return (
        <div className="container text-center" role="dialog" data-testid="status-ok">
            <h2 className="text-success">Twoje nowe dane zostaÅ‚y tymczasowo zapisane ğŸ™‚</h2>
            <p className="mt-2">
                Aby w peÅ‚ni zaktualizowaÄ‡ swoje dane, sprawdÅº swojÄ… skrzynkÄ™
                pocztowÄ… <i className="fa-regular fa-envelope"></i>,
                <br/> gdzie znajdziesz wiadomoÅ›Ä‡ email z linkiem do
                potwierdzenia zmiany <i className="fa-solid fa-check"></i>
            </p>
            <p className="text-info">
                JeÅ›li nie moÅ¼esz znaleÅºÄ‡ tego maila w skrzynce odbiorczej <i className="fa-solid fa-inbox"></i>,
                sprawdÅº folder SPAM
            </p>
            <button className="btn btn-primary mt-2" onClick={() => {
                setIsModalOpen(false);
            }}>OK <i className="fa-solid fa-check"></i>
            </button>
        </div>
    );
}

export default StatusOkModalContent;