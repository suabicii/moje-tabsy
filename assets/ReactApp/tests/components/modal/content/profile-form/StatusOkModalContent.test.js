import React from "react";
import ReactShallowRenderer from "react-test-renderer/shallow";
import StatusOkModalContent from "../../../../../components/modal/content/profile-form/StatusOkModalContent";

it('should correctly render StatusOkModalContent', () => {
    const renderer = new ReactShallowRenderer();
    const setIsModalOpen = jest.fn();

    renderer.render(<StatusOkModalContent setIsModalOpen={setIsModalOpen}/>);

    expect(renderer.getRenderOutput()).toMatchSnapshot();
});