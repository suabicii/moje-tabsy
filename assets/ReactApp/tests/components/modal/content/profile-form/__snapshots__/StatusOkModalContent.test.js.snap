// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render StatusOkModalContent 1`] = `
<div
  className="container text-center"
  data-testid="status-ok"
  role="dialog"
>
  <h2
    className="text-success"
  >
    Twoje nowe dane zostaÅ‚y tymczasowo zapisane ğŸ™‚
  </h2>
  <p
    className="mt-2"
  >
    Aby w peÅ‚ni zaktualizowaÄ‡ swoje dane, sprawdÅº swojÄ… skrzynkÄ™ pocztowÄ… 
    <i
      className="fa-regular fa-envelope"
    />
    ,
    <br />
     gdzie znajdziesz wiadomoÅ›Ä‡ email z linkiem do potwierdzenia zmiany 
    <i
      className="fa-solid fa-check"
    />
  </p>
  <p
    className="text-info"
  >
    JeÅ›li nie moÅ¼esz znaleÅºÄ‡ tego maila w skrzynce odbiorczej 
    <i
      className="fa-solid fa-inbox"
    />
    , sprawdÅº folder SPAM
  </p>
  <button
    className="btn btn-primary mt-2"
    onClick={[Function]}
  >
    OK 
    <i
      className="fa-solid fa-check"
    />
  </button>
</div>
`;
